google-http-client/src/main/java/com/google/api/client/http/OpenCensusUtils.java

```diff
@@ -34,14 +33,12 @@
 import javax.annotation.Nullable;
 
 /**
- * {@link Beta} <br>
  * Utilities for Census monitoring and tracing.
  *
  * @author Hailong Wen
  * @since 1.28
  */
-@Beta
-public class OpenCensusUtils {
+class OpenCensusUtils {
```

Comment:
This looks like a breaking change, even though it's annotated `@Beta`. Is this intentional?

Reply:
Yes it was. This class imports opencensus so its usage I think needs to be guarded by the class loading check. I figured since it was beta it was okay to move it out of the public scope and make it package private. Is that not the case? If we don't shift this to an internal class I don't think we can make the dependencies optional, right?

Reply:
@chingor13 WDYT?

Reply:
"Features marked with the @Beta annotation at the class or method level are subject to change. They might be modified in any way, or even removed, in any major release. You should not use beta features if your code is a library itself (that is, if your code is used on the CLASSPATH of users outside your own control)."